root/static/scripts/event/components/EventEditForm.js

```diff
@@ -115,6 +117,22 @@ function reducer(state: StateT, action: ActionT): StateT {
       newStateCtx.set('showTypeBubble', true);
       break;
     }
+    case 'set-setlist': {
+      fieldCtx.update('setlist', (setlistFieldCtx) => {
+        setlistFieldCtx.set('value', action.setlist);
+        if (isValidSetlist(action.setlist)) {
+          setlistFieldCtx.set('has_errors', false);
+          setlistFieldCtx.set('errors', []);
+        } else {
+          setlistFieldCtx.set('has_errors', true);
+          setlistFieldCtx.set('errors', [
+            l(`Please ensure all lines start with @, * or #,
+                followed by a space.`),
```

Comment:
Untested minor formatting suggestion:

```suggestion
            l(`Please ensure all lines start with
               <code>@</code>, <code>*</code> or <code>#</code>,
               followed by a space.`),
```

Reply:
I planned to commit that to beta separately (instead of part of this PR), and it seems to work: https://gist.github.com/mwiencek/f29c6bb81b2d3425d5b392b279e1afe2

However, after testing it I couldn't notice much difference visually, so not 100% sure if it's worth it. But I can commit that if you'd like.

Reply:
> Edit: In case it wasn't clear, I was copying the existing string from lib/MusicBrainz/Server/Form/Field/Setlist.pm

Okay, I thought that it was a new string. Thanks for having committed this suggestion separately then.

> However, after testing it I couldn't notice much difference visually, so not 100% sure if it's worth it. But I can commit that if you'd like.

This is mainly because we don’t have an appropriate style for displaying `<code>` tags. Additionally to the sans-serif, it should probably have some background color with border radius, light padding and white-space handling; as in GitHub or GitLab for example.